code/modules/cargo/packs.dm

```diff
@@ -626,9 +626,8 @@
 					/obj/item/clothing/under/costume/soviet,
 					/obj/item/clothing/mask/russian_balaclava,
 					/obj/item/clothing/head/helmet/rus_ushanka,
-					/obj/item/clothing/suit/armor/vest/russian_coat,
-					/obj/item/gun/ballistic/rifle/boltaction,
-					/obj/item/gun/ballistic/rifle/boltaction)
+					/obj/item/clothing/suit/armor/vest/russian_coat)
+					//SKYRAT EDIT: REMOVAL - Removed /obj/item/gun/ballistic/rifle/boltaction
```

Comment:
probably should lower the cost of the crate a decent amount considering its main appeal is gone

Reply:
Halfed it